From 2f24d897fdb53156d97556132359dc21f2ef8012 Mon Sep 17 00:00:00 2001
From: Moritz Schulte <mo@g10code.com>
Date: Mon, 16 Feb 2009 00:41:14 +0000
Subject: [PATCH 03/52] 2009-02-16  Moritz  <moritz@gnu.org>

	* pam_poldi.c (pam_sm_authenticate): Include name of the file that
	could not be opened in error message.
---
 src/pam/ChangeLog   | 5 +++++
 src/pam/pam_poldi.c | 3 ++-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/src/pam/ChangeLog b/src/pam/ChangeLog
index a67c860..1c57ce6 100644
--- a/src/pam/ChangeLog
+++ b/src/pam/ChangeLog
@@ -1,3 +1,8 @@
+2009-02-16  Moritz  <moritz@gnu.org>
+
+	* pam_poldi.c (pam_sm_authenticate): Include name of the file that
+	could not be opened in error message.
+
 2009-01-17  Moritz  <moritz@gnu.org>
 
 	* pam_poldi.c (enum opt_ids): New entry: opt_quiet.
diff --git a/src/pam/pam_poldi.c b/src/pam/pam_poldi.c
index 6428215..1cec968 100644
--- a/src/pam/pam_poldi.c
+++ b/src/pam/pam_poldi.c
@@ -325,7 +325,8 @@ pam_sm_authenticate (pam_handle_t *pam_handle,
   if (err)
     {
       log_msg_error (ctx->loghandle,
-		     _("failed to parse configuration file: %s"),
+		     _("failed to parse configuration file '%s': %s"),
+		     POLDI_CONF_FILE,
 		     gpg_strerror (err));
       goto out;
     }
-- 
2.17.0

